app/utils/results.py

```diff
@@ -223,3 +223,90 @@ def add_ip_card(html_soup: BeautifulSoup, ip: str) -> BeautifulSoup:
         # Inserting the element
         ref_element.insert_before(ip_tag)
     return html_soup
+
+
+def check_currency(response: str):
+    """Check whether the results have currency conversion
+
+    Args:
+        response: Search query Result
+
+    Returns:
+        list
```

Comment:
Since the returned list contains a specific set of values that you're expecting in `add_currency_details`, it might be good to document those here. Or it might be better to a key-value dict so that you can reference them by more fitting names (see below)